be/src/storage/vertical_compaction_task.cpp

```diff
@@ -59,6 +59,11 @@ Status VerticalCompactionTask::_vertical_compaction_data(Statistics* statistics)
           "size:$1",
           max_rows_per_segment, column_groups.size());
 
+    // ensure all input rowsets are loaded into memory
+    for (auto& rowset : _input_rowsets) {
```

Comment:
When these _input_rowsets is acquired and when is these rowsets is released?

Reply:
These codes are safeguard.
Since it is still in the mid process of compaction, these _input_rowsets can not be `close`.(we only close the rowsets when compaction is finished or the rowsets are useless) So i think it is safe not to acquire/release these rowsets here?